Chermann-KING/audiophile
# ✨ Home page added with Header, Main and Footer

## Summary of changes

This Pull Request adds a new home page to the project, comprising three main components : `Header`, `Main` and `Footer`.

## Details

- **Header**: The `<Header />` component contains the logo, navigation bar and shopping cart.
- **Main**: The `<Main />` component contains the main content (Hero, Categories, Products and About) of the page.
- **Footer**: The `<Footer />` component contains the logo, navigation bar, useful info, copyright and links to social networks.

### Other changes

- Added the `Home.jsx` file with three components.
- Use of `styled-components` for page styling.
- Use of the `theme` file for colors and typography.

## Why are these changes necessary?

These changes provide a basic structure for the home page, which is essential for further development of the project.

## How do you test the changes?

1. Open the project in your local development environment.
2. Navigate to the home page.
3. You should see the `Header`, `Main` and `Footer` components displayed correctly.
4. Click on the vercel link  (https://audiophile-chermann-king.vercel.app/)

## Associated tickets

- Ticket #1 (Creating a responsive home page)

Thank you for taking the time to check these changes.
